src/components/Preview.js

```diff
@@ -34,6 +34,36 @@ function Preview({ markup, accessibleRoles, elements, dispatch }) {
 
   // TestingLibraryDom?.getSuggestedQuery(highlighted, 'get').toString() : null
 
+  const scripts = useMemo(() => {
+    const container = document.createElement('div');
+    container.innerHTML = markup;
+    const scriptsCollections = container.getElementsByTagName('script');
+    return Array.from(scriptsCollections).map((script) => script.innerHTML);
+  }, [markup]);
+
+  const actualMarkup = useMemo(
+    () =>
+      scripts.length
+        ? scripts.reduce(
+            (html, script) => html.replace(`<script>${script}</script>`, ''),
```

Comment:
This doesn't remove `script` elements that specify the `type` attribute. Type attributes for `script` elements default to `text/javascript`, but it's still a valid attribute to provide. :slightly_smiling_face:

Reply:
Yes, totally right! I am going to fix it!